samples/bot-request-approval/nodejs/Readme.md

```diff
@@ -0,0 +1,95 @@
+---
+page_type: sample
+description: This sample shows a feature where user can send task request to his manager and manager can approve/reject the request in group chat.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+contentType: samples
+createdDate: "6-12-2021 17:00:25"
+---
+
+# Send task request using Universal Adaptive Cards in group chat
+
+This sample shows a feature where:
+1. Requester : Can request for any task approval from manager by intiating request in groupchat using bot command `request` and only requester can edit the request card.
+2. Manager : Can see the request raised by user in the same groupchat with an option of approve or reject.
+3. Others: Other members in the group chat can see the request details only.
+
+Requester:
+
+- Initiated request using bot command `request` in group chat.
+
+  ![Initial Card](Images/InitialCard.png)
+
+- Card will refresh for requester to fill details.
+
+  ![Request Card](Images/CreateTask.png)
+  
+- After submitting the request, requester can edit or cancel the request.
+- Note: Only user who created the card will see the buttons to edit and cancel the request.
+
+  ![Edit/Cancel Card](Images/UserCard.png)
+
+Manager:
+
+- After requester submit the request, manager can approve/reject the request.
+- Note: Only the manager of the request will see the buttons to approve/reject the request.
+
+  ![Approve/Reject Card](Images/ManagerCard.png)
+
+- If manager approves or rejects the request card will be refreshed for all the members in groupchat.
```

Comment:
If manager approves or rejects the request, card will be refreshed for all the members in group chat.